cypress/integration/27-expenses.create.test.js

```diff
@@ -22,10 +22,9 @@ describe('new expense when logged out', () => {
   it('requires to login to submit an expense', () => {
     cy.visit('/testcollective/expenses/new');
     cy.get('.CreateExpenseForm').contains('Sign up or login to submit an expense');
-    cy.get('.inputField.email input').type('testuser+admin@opencollective.com');
-    cy.get('.login').click();
+    cy.get('#email').type('testuser+admin@opencollective.com');
+    cy.get('[data-cy="signin-btn"]').click();
     cy.wait(300);
-    cy.get('.inputField.description', { timeout: 5000 });
```

Comment:
Why removing this?

Reply:
It was failing on my local machine so just removed it to see if all tests were passing further. Added it again.

Reply:
There is no need of using it actually. After we sign in we are redirected to http://localhost:3000 where there is no input field present so it gives an error.